tidy3d/web/core/http_util.py

```diff
@@ -129,6 +131,10 @@ def wrapper(*args, **kwargs):
         if resp.status_code != ResponseCodes.OK.value:
             if resp.status_code == ResponseCodes.NOT_FOUND.value:
                 return None
+            if resp.status_code == ResponseCodes.UNPROCESSABLE.value and resp.text:
+                result = resp.json()
+                if str(ResponseCodes.TASK_REPEATED.value) == result["code"]:
```

Comment:
I'm worried about obscure errors happening if `"code"` or `"detail"` are not found in `result`. Is there a way to handle this more gracefully?  maybe
```py
if any(key not in result for key in ("code", "detail")):
    # do task repeated logic
else:
    # not sure, either raise a new WebError about missing keys or just continue?
```

```